Makefile

```diff
@@ -131,6 +133,9 @@ pre-commit: fmt lint
 init:
 	@bin/init.sh
 
+# init.sh downloads envoy
+${ISTIO_BIN}/envoy: init
```

Comment:
Well, if you're going with each tool having a variable, 
ENVOY ?= ...

Reply:
In this case i was thinking of envoy as more like a binary artifact that's consumed by the build rather than something that's run by the build.  I think one of the docker images tried to ADD the file in their dockerfile and this was the best I came up with to record the dependency in the make rule and educate make on whether that dependency comes from.